### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by songxianj**
Signed-off-by: JIANG Songxian [songxian.jiang@zilliz.com](mailto:songxian.jiang@zilliz.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,29 +31,29 @@
...
-You can prototype your pipeline with our Pythonic API, and use Towhee to
+We provide several hundred models, algorithms, and transformations that can be used as standard pipeline building blocks.
+You can use Towhee's Pythonic API to build a prototype of your pipeline and
 automatically optimize it for production-ready environments.
 
-:art:&emsp;**Various Modalities:** We support data processing on different modalities, such as images, videos, text, audio, molecular structures, etc.
+:art:&emsp;**Various Modalities:** Towhee supports data processing on a variety of modalities, including images, videos, text, audio, molecular structures, etc.
 
-:mortar_board:&emsp;**SOTA Models:** We provide SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, 140+ model architectures, 700+ pretrained models. These include BERT, CLIP, ViT, SwinTransformer, MAE, data2vec, etc.
+:mortar_board:&emsp;**SOTA Models:** Towhee provides SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, and 140+ model architectures, including BERT, CLIP, ViT, SwinTransformer, MAE, and data2vec. All pretrained and 700+ more others to choose from.
```
Comment (filip-halt) [inline README.md:40]: Towhee provides SOTA models across 5 fields (CV, NLP, Multimodal, Audio, Medical), 15 tasks, and 140+ model architectures. These include  BERT, CLIP, ViT, SwinTransformer, MAE, and data2vec, all pretrained and ready to use.

File: README.md [hunk 2]
```diff
@@ -31,29 +31,29 @@
...
 
-:package:&emsp;**Data Processing:** Towhee also provides traditional data processing methods that can be used together with neural network models to help you build practical data processing pipelines. Video decoding, audio slicing, frame sampling, feature vector dimension reduction, model ensemble, and database operations are a small sample of the different operators we provide.
+:package:&emsp;**Data Processing:** Towhee also provides traditional methods alongside with neural network models to help you build practical data processing pipelines. We have a rich pool of operators available, such as video decoding, audio slicing, frame sampling, feature vector dimension reduction, model ensembles, and database operations.
```
Comment (filip-halt) [inline README.md:42]: Towhee also provides traditional methods alongside neural network models to help you build practical data processing pipelines. We have a rich pool of operators available, such as video decoding, audio slicing, frame sampling, feature vector dimension reduction, ensembling, and database operations.

File: README.md [hunk 3]
```diff
@@ -31,29 +31,29 @@
...
 
-:snake:&emsp;**Pythonic API:** Towhee includes a pythonic method-chaining API for describing custom data processing pipelines. We also support schemas, making processing unstructured data as easy as handling tabular data.
+:snake:&emsp;**Pythonic API:** Towhee includes a Pythonic method-chaining API for describing custom data processing pipelines. We also support schemas, which make processing unstructured data as easy as handling tabular data.
```
Comment (filip-halt) [inline README.md:44]: :snake:&emsp;**Pythonic API:** Towhee includes a Pythonic method-chaining API for describing custom data processing pipelines. We also support schemas, which makes processing unstructured data as easy as handling tabular data.

File: README.md [hunk 4]
```diff
@@ -85,14 +84,14 @@ automatically optimize it for production-ready environments.
...
-Towhee requires Python 3.6+. Towhee can be installed via `pip`:
+Towhee requires Python 3.6+. You can install Towhee via `pip`:
 
 ```bash
 % pip install -U pip  # if you run into installation issues, try updating pip
 % pip install towhee
 ```
 
-Try your first Towhee pipeline. In this example, we show how to create a CLIP-based cross modal retrieval pipeline within 15 lines of code.
+Try your first Towhee pipeline. For example, create a CLIP-based cross modal retrieval pipeline within 15 lines of code.
```
Comment (filip-halt) [inline README.md:94]: Try your first Towhee pipeline. For example, create a CLIP-based cross modal retrieval pipeline with 15 lines of code.

File: README.md [hunk 5]
```diff
@@ -117,23 +116,23 @@ results = (
...
 
-Learn more examples from [Towhee bootcamp](https://codelabs.towhee.io/)
+Learn more examples from the [Towhee Bootcamp](https://codelabs.towhee.io/).
 
 ## Core Concepts
 
 Towhee is composed of four main building blocks - `Operators`, `Pipelines`, `DataCollection API` and `Engine`.
 
-- __Operator__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
+- __Operators__: An operator is a single building block of neural data processing pipelines. Different implementations of operators are categorized by tasks, with a standard task interface. An operator can be a deep learning model, a data processing method, or a Python function.
```
Comment (filip-halt) [inline README.md:125]: - __Operators__: An operator is a single building block of a neural data processing pipeline. Different implementations of operators are categorized by tasks, with each task having a standard interface. An operator can be a deep learning model, a data processing method, or a Python function.

File: README.md [hunk 6]
```diff
@@ -117,23 +116,23 @@ results = (
...
 
-- __Pipeline__: A pipeline is composed of several operators. Operators are connected together as a DAG(directed acyclic graph) to create complex functionalities, such as embedding feature extraction, data tagging, cross modal data understanding, etc.
+- __Pipelines__: A pipeline is composed of several operators. Operators are connected together as a DAG (directed acyclic graph) to create complex functionalities, such as embedding feature extraction, data tagging, and cross modal data understanding.
```
Comment (filip-halt) [inline README.md:127]: - __Pipelines__: A pipeline is composed of several operators interconnected in the form of a DAG (directed acyclic graph). This DAG can direct complex functionalities, such as embedding feature extraction, data tagging, and cross modal data analysis.

File: README.md [hunk 7]
```diff
@@ -117,23 +116,23 @@ results = (
...
 
-- __DataCollection__: A pythonic and method-chaining style API that for building custom pipelines. Pipelines defined by DataColltion API can be either run on notebook locally for fast prototyping, or converting to image docker with end-to-end optimization for production-ready environments. 
+- __DataCollection API__: A Pythonic and method-chaining style API for building custom pipelines. Pipelines defined by DataColltion API can run on laptops locally for fast prototyping, or can be converted to docker images with end-to-end optimization for production-ready environments.
```
Comment (filip-halt) [inline README.md:129]: - __DataCollection API__: A Pythonic and method-chaining style API for building custom pipelines. A pipeline defined by the DataColltion API can be run locally on a laptop for fast prototyping and then be converted to a docker image, with end-to-end optimizations, for production-ready environments.

File: README.md (hunk 1 above)
Comment (songxianj) [reply README.md]: Thank you for all the comments. It was especially hard for me to figure out the best way to rephrase the "pretrained" bit. Changes have been made in a new push.

**Pull request merged**